test/client/fixtures/sandbox/event/event-test.js

```diff
@@ -314,6 +315,15 @@ test('should not wrap invalid event handlers (GH-1251)', function () {
         });
 
         strictEqual(listeningCtx.getEventCtx(target, 'click').wrappers.length, storedHandlerWrappersCount);
+
+        handlers.forEach(function (handler) {
```

Comment:
Why do we need a `try/catch` statement?

Reply:
It is a test for invalid event handlers. See the line above: `var handlers = [void 0, 1, null, 'str', {}];`. The test adds some wrong event handlers, but does not remove them, so it leads to problems with next texts. I've added code which removes handlers in the same manner as code which adds handlers